code/modules/research/xenoarchaeology/finds/finds.dm

```diff
@@ -586,9 +586,9 @@
 /datum/find/laser/spawn_item()
 
 	var/gun_base = pickweight(list(
-		/obj/item/weapon/gun/energy/laser			=	70,		//70% chance to be a normal gun
-		/obj/item/weapon/gun/energy/laser/captain	=	15,		//15% chance to be self-recharging
-		/obj/item/weapon/gun/energy/bison			= 	5,		//5% chance to be pump-charge
+		/obj/item/weapon/gun/energy/laser/alien				=	70,		//70% chance to be a normal gun
+		/obj/item/weapon/gun/energy/laser/captain/alien	=	15,		//15% chance to be self-recharging
+		/obj/item/weapon/gun/energy/bison/alien			= 	5,		//5% chance to be pump-charge
```

Comment:
These do not add to 100, so the comments are wrong, actual chances are 77,7%, 16,6% and 5,5%.

Change the numbers so they add to 100

Reply:
[He knows](https://github.com/vgstation-coders/vgstation13/pull/27646/files#r484474560)

Reply:
fixed the numbers now,  I forgot to do this earlier